CITATION.cff

```diff
@@ -243,7 +243,7 @@ authors:
   - given-names: Fabian
     family-names: Pedregosa
     website: https://github.com/fabianp
-    affiliation: '@google, Geneva, Switzerland'
+    affiliation: "@google, Geneva, Switzerland"
```

Comment:
I didnt realize that these quotes got automatically added by black. I think it's ok keep them